src/common/ad_mem.c

```diff
@@ -1986,7 +1992,7 @@ arena_reserve_grp(struct ad_arena *arena, daos_size_t size, int *pos,
 		return -DER_NOMEM;
 
 	/* find an unused one */
-	for (grp_idx = 0; grp_idx < ARENA_GRP_MAX; grp_idx++) {
+	for (grp_idx = arena->ar_last_grp; grp_idx < ARENA_GRP_MAX; grp_idx++) {
```

Comment:
This works because group reset is disabled for now right? otherwise there might be some groups before ar_last_grp reset and freed.

Reply:
I see, thanks.